### DISCUSSION TITLE
descriptionProps

### DISCUSSION THREAD

**Discussion by howlwindy** [tusen-ai/naive-ui#2908] [Q&A] [answered]
请问暴露出的组件和props的命名基本都是：
组件：`NComp`
props：`CompProps`

而descriptions组件的命名是这样：
组件：`NDescriptions`
props：`DescriptionProps`

为什么`NDescriptions`的props命名是`DescriptionProps`，而不是`DescriptionsProps`？

十分感谢

**Comment by 07akioni** [comment, marked as answer]
我写错了，会改掉

**Reply by 07akioni** [reply]
改了，之后会发布
